subsys/bluetooth/services/Kconfig.bms_srv

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menuconfig BT_GATT_BMS_SRV
+	bool "Bond Management Service"
```

Comment:
I think we should put a restriction on not having BT_SMP_OVERWRITE_OLDEST feature enabled.